tests/README.md

```diff
@@ -228,7 +239,9 @@ markers = [
     ... # <-- add new marker
 ```
 
-- Execute the test suite by running the following command and replacing marker `aws_logging_rules` with newly defined marker
+- Execute the test suite by running the following command and replacing marker `aws_logging_rules` with newly defined
+  marker
```

Comment:
revert

Reply:
this is automatically formatted by our pre-commit hook to enforce line length (it won't affect the actually markdown output)